### PR TITLE
New Colorbox plugin & Insert class enhancements

### PR DESCRIPTION
**Pull request by WebDesignWorx**

### CODE REVIEW

File: wbce/framework/I.php [hunk 1]
```diff
@@ -283,7 +283,7 @@ public static function addUrlToken($sToken = "", $sURL = "") {
      */
     public static function doFilter($Content) {
        $result = I::getInstance()->doFilter($Content);
-       I::$oInstance = NULL;
+       #I::$oInstance = NULL;
        return $result;
```
Comment (mrbaseman) [inline wbce/framework/I.php]: With the more elegant solution we could revert this function to the original oneliner

File: wbce/framework/Insert.php [hunk 2]
```diff
@@ -1258,6 +1258,13 @@ public function doFilter($sContent)
                 1
             );
         }
+        unset(
+            $this->_JsQueue,
+            $this->_CssQueue,
+            $this->_HtmlQueue,
+            $this->_TitleQueue,
+            $this->_MetaQueue
+        );
```
Comment (mrbaseman) [inline wbce/framework/Insert.php]: This is much more elegant +1

File: wbce/framework/I.php (hunk 1 above)
Comment (WebDesignWorx) [reply wbce/framework/I.php]: You're absolutely right, Martin.

File: wbce/framework/Insert.php (hunk 2 above)
Comment (WebDesignWorx) [reply wbce/framework/Insert.php]: I actually liked your solution more, but we need to maintain the same functionality in both classes. Or at least, the Insert class shouldn't have less functionality than its static facade class I.

Regards, 
Christian

**Pull request merged**